### ISSUE TITLE
Add an explicit way to get extra retrieved files in the calculations

### ISSUE THREAD

**Issue by JPchico** [aiidaplugins/aiida-lammps#88]
As far as I know only the files explicitly stated in the `retrieve_list`  are copied to the repository. However, specially for the raw calculation one might want to get several more files (trajectories, etc) which are not gotten by default. Thus, adding an entry in the settings where such a list of files is given would help with such an issue.

@sphuber do you agree? Or is there another method of doing this I'm unaware of?

**Comment by sphuber** [comment]
Yep, I agree that that might be useful, and indeed, traditionally, adding an `additional_retrieve_list` to a `settings` input is one way of accomplishing that.

But I wonder if users that want to run a raw script would not simply be better off using `aiida-shell`. All of this is already available and it would look like:
```
from aiida_shell import launch_shell_job

results, node = launch_shell_job(
    'lmp',
    arguments='-in {script} -log output.log',
    nodes={
        'script': '/path/to/script.lmp'
    },
    outputs=['stdout', 'output.log'],
    metadata={
        'options': {
            'parser_name': 'lammps.raw'
        }
    }
)
```
They can add any output files they want retrieved to the `outputs` list.

**Comment by JPchico** [comment]
Sure I think that the `aiida-shell` is great for this case. But it might be good to have it in the raw and base calculations just in case, I'll quickly add it in after #77 is merged.

**Comment by JPchico** [comment]
Closed via #89
